scripts/core/editor3/directive.js

```diff
@@ -111,6 +115,14 @@ class Editor3Directive {
 
         const store = createStore(this);
 
+        $scope.$watch('vm.value', (newValue, oldValue) => {
```

Comment:
imo that's confusing, it shouldn't be editable right?
wouldn't it work with just `ng-bind-html` ? why is it simpler with editor?